src/models/StyledNativeComponent.js

```diff
@@ -150,6 +143,24 @@ export default (constructWithOptions: Function) => {
     }
   }
 
+  nativeMethodKeys.forEach(methodKey => {
+    // $FlowFixMe
+    BaseStyledNativeComponent.prototype[methodKey] = function nativeProxy(...args) {
+      if (this.root !== undefined) {
+        this.root[methodKey](...args)
+      } else {
+        const { displayName } = this.constructor
+
+        // eslint-disable-next-line no-console
+        console.warn(
+          `${methodKey} was called on a Styled Component wrapping a stateless functional component. ` +
+          'In this case no ref will be stored, and instead an innerRef prop will be passed on.\n' +
+          `Check whether the stateless functional component is passing on innerRef as a ref in ${displayName}.`,
```

Comment:
Yes, we can assume that when the this.root isn't set, that we couldn't set it because the underlying component is a SFC, and that the user hasn't passed on innerRef :)